### DISCUSSION TITLE
Building on MacOS M1 completes but SS is damaged

### DISCUSSION THREAD

**Discussion by Hr46ph** [supermerill/SuperSlicer#3995] [General]
I get:
```
[9/9] Generating MacOS app...done
Creating DMG Image for distribution...done
```
and an `ls` output with (among other files) these:
```
-rw-r--r--  1 noob staff  40M Dec  8 17:23  SuperSlicer.dmg
-rw-r--r--  1 noob staff  61M Dec  8 17:41  SuperSlicer.tar
-rw-r--r--  1 noob staff  41M Dec  8 17:41  temp.dmg
```
I ran the build script with `-i` at first, when I got files with a stop sign overlaid on the application icon (and says its damaged upon running), I tried building with `-a -i`, to specify ARM. Same result.

I did a fresh git clone of the master branch.

Is it not building at this point?

**Comment by supermerill** [comment]
i'm not sure.
Sometimes, it can fail to build the app but will still continue building the dmg. check if there's not an error, and if the superslicer app inside the dmg is here.

**Comment by Hr46ph** [comment]
It doesn't seem that the app is inside. Don't worry about it though, I'll wait for your 2.7 based release.

**Comment by pauldotknopf** [comment]
Is the 2.7 release expected to contain a pre-built binary for the M1 Mac (arm64)?

**Reply by supermerill** [reply]
I'll work on arm build workflow after 2.7 merge is in debug.
With the new github arm runner, the pull request that fixes the signing of the dmg, you should have good dmg, and even maybe an universal one.
